app/Services/Manager.php

```diff
@@ -189,19 +204,31 @@ public function collectContestInfo($data)
      *
      * @return object $signups
      */
-    public function getUserSignup($id, $campaign = null, $campaignRun = null)
+    public function getUserSignup($id, $campaign = null, $campaignRun = null, $getMultiple = false)
     {
         $signups = $this->northstar->getUserSignups($id, $campaign);
 
+        if ($getMultiple) {
+            $multipleSignups = [];
+        }
```

Comment:
I think either we should rename this to `getUsersSignups()` (plural) or maybe separate out into two different functions where one returns multiple signups and one returns a single signup. From a pure readability POV it seems weird to call `getUserSignup` but have to designate that you want multiple signups. But if we just pluralize it from the jump it makes sense that if you only provide one `id` you will only get one signup.